display.h

```diff
@@ -207,7 +207,8 @@ enum s9x_getdirtype
 	IPS_DIR,
 	BIOS_DIR,
 	LOG_DIR,
-	LAST_DIR
+	LAST_DIR,
```

Comment:
...I don't even remember why I did this to begin with. I probably did it a little too quick.